components/inline/test/inline.test.js

```diff
@@ -29,32 +30,25 @@ describe('inline', () => {
 		it('should have internal contents', async () => {
 			const actualElement = getNewElement();
 			await waitNextTask();
-			expect(actualElement.shadowRoot.innerHTML).to.equalSnapshot();
+			expect(actualElement.shadowRoot.firstElementChild.innerHTML).to.equalSnapshot();
 		});
 	});
 
-	describe('API', () => {
-		it(`should set template fit`, async () => {
-			const actualElement = getNewElement();
-			actualElement.template = "fit";
-			actualElement.style.width = "1300px";
-			await waitNextTask();
-			const { shadowRoot: { firstElementChild: slot } } = actualElement;
-			const assignedElements = slot.assignedElements();
-			const [childEl] = assignedElements;
-			await waitNextTask();
-			expect(childEl.clientWidth).to.equal(307);
-		});
-		it(`should set template fill`, async () => {
-			const actualElement = getNewElement();
-			actualElement.template = "fill";
-			actualElement.style.width = "1300px";
-			await waitNextTask();
-			const { shadowRoot: { firstElementChild: slot } } = actualElement;
-			const assignedElements = slot.assignedElements();
-			const [childEl] = assignedElements;
-			await waitNextTask();
-			expect(childEl.clientWidth).to.equal(165);
+	describe('default values', () => {
+		it('should have the default values', async () => {
+			const COMPONENT_PROPERTIES = {
+				columnBasis: 'sm', columnSpacing: 'md', inlineGutters: 'md'
```

Comment:
are we checking properties attributes sync? maybe important especially when the names aren't identical